README.md

```diff
@@ -1,5 +1,6 @@
 # Sneakers
 
+[![Build Status](https://travis-ci.org/jondot/sneakers.svg?branch=master)](https://travis-ci.org/jondot/sneakers)
```

Comment:
Oh, you're totally right. My apologies, I missed this. As the PR is already closed, how should we proceed? New PR?